src/metatensor/models/experimental/pet/tests/test_functionality.py

```diff
@@ -44,3 +46,81 @@ def test_prediction_subset():
         evaluation_options,
         check_consistency=True,
     )
+
+
+def test_per_atom_predictions_functionality():
+    """Tests that the model can predict on a subset
+    of the elements it was trained on."""
+
+    capabilities = ModelCapabilities(
+        length_unit="Angstrom",
+        atomic_types=[1, 6, 7, 8],
+        outputs={
+            "energy": ModelOutput(
+                quantity="energy",
+                unit="eV",
+                per_atom=True,
```

Comment:
if PET can now do per-atom predictions, this should be set in the train script as well!

Reply:
Thank you, well noticed!